### ISSUE TITLE
getRawSchemalessAttributes() must be of the type array, string returned

### ISSUE THREAD

**Issue by fede91it** [spatie/laravel-schemaless-attributes#45]
When a schemaless column from db is set to NULL instead of `{}`, on scout:import it results in that error:

Return value of Spatie\SchemalessAttributes\SchemalessAttributes::getRawSchemalessAttributes() must be of the type array, string returned

**Comment by freekmurze** [comment]
Feel free to submit a PR with tests that fixes this.

**Comment by chinleung** [comment]
> When a schemaless column from db is set to NULL instead of `{}`, on scout:import it results in that error:
> 
> Return value of Spatie\SchemalessAttributes\SchemalessAttributes::getRawSchemalessAttributes() must be of the type array, string returned

Isn't this expected behaviour though? Your database value is most likely set to **NULL** as a **string**. Can you confirm for this? 🤔

**Comment by terranc** [comment]
I have the same problem at PHP v7.2.14

**Comment by securit** [comment]
Yes this issue is back because the function is type hinted to an array, 
```
    protected function getRawSchemalessAttributes(): array
```
but it calls the return function fromJson which returns a mixed type.

**Comment by securit** [comment]
This problem appears to be an issue still, particularly for those of us using Postgres when you have a null object, ie {}::json.

**Comment by BinaryKitten** [comment]
move to close: has this been resolved by #60 ?
(came here via good first issue)
